config.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
```

Comment:
you can ignore this file. I think this was already merged in master, if you do merge with master should be gone